scripts/linters/general_purpose_linter.py

```diff
@@ -259,10 +259,11 @@
         'excluded_dirs': ()
     },
     {
-        'regexp': re.compile(r'import \{.*\} from \'lodash\''),
+        'regexp': re.compile(r'import (\{.*\}|\_) from \'lodash\''),
         'message': (
-            'Please do not use "import { someFunction } from \'lodash\'". '
-            'Use "import someFunction from \'lodash/someFunction\'" instead.'),
+            'Please do not use "import { someFunction } from \'lodash\'" a '
```

Comment:
Can you please change the existing test to be more strict for this? (or add multiple tests)

Reply:
Done.